### PR TITLE
Added Weekly Meeting Details

### LINKED ISSUE #22: [Docs] README: add weekly meeting details
**Current State:**
A weekly meeting focuses filters (and related topics) found in this repo.

**Desired State:**
Ideally, this readme has a description of and invitation to the weekly meeting (details -https://meet.layer5.io)

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by includestdlibrary**
Signed-off-by: Ritvik Shukla <ritvikshukla261@gmail.com>

**Description**

This PR Fixes #22 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by includestdlibrary** [comment]
@sayantan1413  please check now is it ok?  Sorry I'm disturbing you and lee many times in just writing some lines.

**Comment by sayantan1413** [comment]
No no @includestdlibrary we are happy to help you here. But I have one think to say you that whenever you do a change everytime you do not have to close and then open a new PR as it will become difficult for the reviewers to check your previous reviews or comments. So in a PR you can do as many changes as you want so just don't change your branch. Commit from the same branch to the same PR you will see that a new commit get added to your previous commit which contains the latest changes. So for the lesson I taught you I have an exercise for you. You can see that now it is `7:30pm-` but instead you change it to `7:30pm - `. 
(just make the changes and commit from the same branch and do not close this pr).

**Comment by includestdlibrary** [comment]
> 
> 
> No no @includestdlibrary we are happy to help you here. But I have one think to say you that whenever you do a change everytime you do not have to close and then open a new PR as it will become difficult for the reviewers to check your previous reviews or comments. So in a PR you can do as many changes as you want so just don't change your branch. Commit from the same branch to the same PR you will see that a new commit get added to your previous commit which contains the latest changes. So for the lesson I taught you I have an exercise for you. You can see that now it is `7:30pm-` but instead you change it to `7:30pm - `.
> (just make the changes and commit from the same branch and do not close this pr).

Done, Please check if it is fine, And thank you so much for helping. I am very happy to be a part of this community :)

**Review by sayantan1413 — APPROVED** [review]
Looks good to me. Great work @includestdlibrary.

**Comment by sayantan1413** [comment]
> > No no @includestdlibrary we are happy to help you here. But I have one think to say you that whenever you do a change everytime you do not have to close and then open a new PR as it will become difficult for the reviewers to check your previous reviews or comments. So in a PR you can do as many changes as you want so just don't change your branch. Commit from the same branch to the same PR you will see that a new commit get added to your previous commit which contains the latest changes. So for the lesson I taught you I have an exercise for you. You can see that now it is `7:30pm-` but instead you change it to `7:30pm - `.
> > (just make the changes and commit from the same branch and do not close this pr).
> 
> Done, Please check if it is fine, And thank you so much for helping. I am very happy to be a part of this community :)

My pleasure to help you. :)

File: README.md [hunk 1]
```diff
@@ -147,3 +147,15 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
...
 **Open Source First**
 <p>Our projects establish industry standards and enable service developers, owners, and operators with repeatable patterns and best practices for managing all aspects of distributed services. Our shared commitment to the open source spirit push the Layer5 community and its projects forward.</p>
+
+### Weekly Meeting Details
+
+**Weekly on Monday**
+
+Topic : WebAssembly Filters Meeting
+
+Time : 7:30pm – 8:15pm (IST)
```
Comment (leecalcote) [inline README.md:157]:
```suggestion
Time: 7:30pm  IST / 3:00pm UK / 9:00am Central 
```

**Review by leecalcote — APPROVED** [review]
Looking good. Thanks, gents.

**Pull request merged**